src/message/__tests__/messageActionSheet-test.js

```diff
@@ -157,7 +144,7 @@ describe('constructHeaderActionButtons', () => {
 
     const subscriptions = deepFreeze([
       {
-        ...baseBackgroundData.subscriptions[0],
+        ...eg.subscription,
```

Comment:
Yeah -- specifically, I left it explicit here because it's part of what this test is about.

If it were omitted, then looking at this test you wouldn't be able to tell whether the test actually tests the case it's meant to test unless you knew that `eg.subscription` definitely has `in_home_view: true`. That is the logical default, and so you could perhaps figure that it ought to be the default... but even to know it's the logical default, you have to know, and think about, some context about what it *means*.

Whereas when it's explicitly here, then you can look at this test code very locally, and think through it at just a very straightforward step-by-step level, and see that it does exactly what it says it does.